doc/mgr/dashboard.rst

```diff
@@ -1370,6 +1370,37 @@ something like this::
     $ ceph config reset 11
 
 
+Centralized Logging in Ceph
+"""""""""""""""""""""""""""
+
+Ceph-Dashboard now provides centralized logging using Loki & Promtail.
+For this, two new services are introduced in the service list (loki & promtail).
+These services are not deployed by default. To enable the centralized logging -
+
+1. Create the loki service on any particular host using Create services option.
+
+2. Similarly create the promtail service which will be by default deployed 
+   on all the running hosts.
+
+3. To see debug-level messages as well as info-level events, run the following command via CLI::
+
+    $ ceph config set mgr mgr/cephadm/log_to_cluster_level debug
+
+4. To enable logging to files, run the following commands via CLI::
+
+    $ ceph config set global log_to_file true
+
+    $ ceph config set global mon_cluster_log_to_file true
+
+5. Click on the Daemon Logs tab under Cluster -> Logs.
+
+6. You can find some pre-defined labels there on clicking the Log browser button such as filename,
+   job etc that can help you query the logs at one go.
+
+7. You can query the logs with LogQL for advanced search and perform some
+   calculations as well - https://grafana.com/docs/loki/latest/logql/.
```

Comment:
What about moving this to the Cephadm guide, in the monitoring services section: https://docs.ceph.com/en/latest/cephadm/services/monitoring/